CDCgov/prime-reportstream
# lint/prettier tool wasn't really working on all files

Windows didn't like:
 "eslint 'src/**/*.{js,ts,jsx,tsx}'"
Mac doesn't like:
 "eslint src/**/*.{js,ts,jsx,tsx}"
But this combo seems to work:
 "eslint \"src/**/*.{js,ts,jsx,tsx}\""

Steps to test:
1. just switch to this branch
2. refresh your npm command list in webstorm
3. Make some style mistake in any file (e.g. the includes sections)
4. run lint command to see if it fixes the issue.
